components/widgets/resources/src/main/resources/resources/uicomponents/widgets/ageOfOnset.js

```diff
@@ -0,0 +1,100 @@
+var PhenoTips = (function(PhenoTips) {
+  // Start PhenoTips augmentation
+  var widgets = PhenoTips.widgets = PhenoTips.widgets || {};
+
+  widgets.AgeOfOnset = Class.create({
+    // The age until which the specified HPO term applies
+    ageMonthsToHpoTerm: [
+      [0.92008, "HP:0003623"],
```

Comment:
DurationField uses 0.03286 months/day ([source](https://github.com/phenotips/phenotips/blob/d4d624b3d280fada8e06a2e58a04a7d408f35cc0/components/widgets/resources/src/main/resources/resources/uicomponents/widgets/durationField.js#L68)), so I just multiplied this by 28. The calculation would actually use 365.25 (average days/year considering leap years), which would give 365.25/12=30.4375, 28/30.4375 = 0.91992.

There is some precision lost in the DurationField's months/day and as a result some imprecision in "28 days == 0.92008 months" but it makes sense to match the DurationField's value here rather than the more precise value.